### DISCUSSION TITLE
Add support for setQueriesData

### DISCUSSION THREAD

**Discussion by gionathas** [trpc/trpc#3036] [Ideas]
Would be great to have access to [queryClient.setQueriesData](https://tanstack.com/query/v4/docs/reference/QueryClient#queryclientsetqueriesdata) helper method, from the tRPC [useContext](https://trpc.io/docs/v10/useContext#usage).

In this way it would be possible to update cached data of multiple queries by using filter function or partially matching the query key.

**Comment by brian-bourdon** [comment]
I approve this, would be very useful.

**Comment by KATT** [comment]
Make a feature request instead please ☺

**Reply by KATT** [reply]
(As in please use the feature template in issues)
